.github/workflows/ci.yml

```diff
@@ -71,6 +71,21 @@ jobs:
     - name: Perform CodeQL Analysis
       uses: github/codeql-action/analyze@v2
 
+    - name: Aqua Security Trivy
+      uses: aquasecurity/trivy-action@0.17.0
+      with:
+        scan-type: 'fs'
+        scan-ref: '.'
+        format: 'sarif'
+        output: 'trivy-results.sarif'
+        exit-code: 1
+        severity: UNKNOWN,LOW,MEDIUM,HIGH,CRITICAL
```

Comment:
Do we want to error out if there's a low severity action found?

Reply:
As long as it still notifies us of the low severity action, I'm okay with removing it from this list.

Reply:
it looks like there isn't a good way to only exit on High and Critical while still alerting on the lower severities. We could have two separate steps, one to make a report with all vulnerabilities and one that exits on severe vulnerabilities. 

But for now, I think we can trial having all vulnerabilities error out and see how it goes